src/components/PhotoZoom.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useMemo, useEffect, useState, useCallback } from "react"
+import useScrollBlock from "../hooks/useScrollBlock"
+import { TransformWrapper, TransformComponent } from "react-zoom-pan-pinch"
+
+import { BaseProps } from "../data/types"
+interface PhotoZoomProps extends BaseProps {
+  photo: string
+  isLoading: boolean | undefined
+  exit: () => void
+  isTransitioningIn: boolean
+  isTransitioningOut: boolean
+}
+
+function PhotoZoom({
+  photo,
+  isLoading,
+  exit,
+  isTransitioningIn,
+  isTransitioningOut,
+}: PhotoZoomProps) {
+  const [blockScroll, allowScroll] = useScrollBlock()
```

Comment:
This seems unused at the moment.

Reply:
yes, I am going to keep it on the hooks module in case is needed in the future